File: 백준/Bronze/2908. 상수/상수.py
a, b = map(int, input().split())

strA = str(a)
strB = str(b)

listA = list(strA)
listB = list(strB)

listA.reverse()
listB.reverse()

resultA = int("".join(listA))
resultB = int("".join(listB))

if listA > listB:
   print(resultA)
else:
   print(resultB)

File: 백준/Bronze/2908. 상수/상수.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

let [A, B] = input[0].split(' ').map(Number);

let reverseA = Number(String(A).split('').reverse().join(''));
let reverseB = Number(String(B).split('').reverse().join(''));

if (reverseA > reverseB) {
  console.log(reverseA);
} else {
  console.log(reverseB);
}

